anda/lib/gsettings-qt/gsettings-qt.spec

```diff
@@ -0,0 +1,63 @@
+%global forgeurl https://gitlab.com/ubports/development/core/gsettings-qt
+%global commit d5e002d7e0bce46c315bcc99a44a8bd51f49f488
+%forgemeta
+
+Name:       gsettings-qt
+Version:    0.3
```

Comment:
Forgot to add comment, same issue: there's no 0.3 in gitlab but only 0.2, and I've searched repology and none is on 0.3;

However, I do have to say it's quite outdated so I think maybe we can just straight up add the commit to the version (have seen other repos done that)